overlord/configstate/configstate.go

```diff
@@ -30,10 +30,11 @@ import (
 	"github.com/snapcore/snapd/overlord/hookstate"
 	"github.com/snapcore/snapd/overlord/snapstate"
 	"github.com/snapcore/snapd/overlord/state"
+	"github.com/snapcore/snapd/snap"
 )
 
 func init() {
-	snapstate.Configure = Configure
+	snapstate.Configure = configure
```

Comment:
Do we really want this to be configure and not Configure?   If we do, then it would be better to have a different function name that justifies that difference, otherwise it feels like an oversight.

Reply:
we do because it's used in snapstate also when the snap is not yet installed but to be installed,  I agree maybe a different name is better but a bit unclear what's best ? InstallingConfigure?  ConfigureForInstall ?